README.md

```diff
@@ -533,6 +571,63 @@ project.getMetadata(function(err, metadata) {
 ```
 
 
+## Google Cloud Speech (Limited Preview)
+
+> **This is a Limited Preview release of Google Cloud Speech.** This feature is not covered by any SLA or deprecation policy and may be subject to backward-incompatible changes.
+
+- [API Documentation][gcloud-speech-docs]
+- [Official Documentation][cloud-speech-docs]
+
+```js
+var gcloud = require('gcloud');
+
+// Authorizing on a per-API-basis. You don't need to do this if you auth on a
+// global basis (see Authorization section above).
+
+var speech = gcloud.speech({
+  projectId: 'my-project',
+  keyFilename: '/path/to/keyfile.json'
+});
+
+// Detect the speech in an audio file.
+speech.detect('./audio.raw', {
+  encoding: 'LINEAR16',
+  sampleRate: 16000
+}, function(err, result) {
+  // result = {
+  //   transcript: 'how old is the Brooklyn Bridge',
+  //   confidence: 0.97234234234
+  // ]
+});
+```
+
+```js
+// Detect the speech in an audio file stream.
+var initialRequest = {
+  encoding: 'LINEAR16',
+  sampleRate: 16000,
+  continuous: true
```

Comment:
Does this field distinguish the first type of request (all in one go) from the second (streaming) ? If so, this might be better to separate out as another method.